packages/core/src/amazonq/webview/ui/main.ts

```diff
@@ -582,7 +587,8 @@ export const createMynahUI = (
                 eventId,
                 codeBlockIndex,
                 totalCodeBlocks,
-                messageUserIntentMap.get(messageId) ?? undefined
+                responseMetadata.get(messageId)?.[0] ?? undefined,
```

Comment:
We should add these to `ViewDiff` and `AcceptDiff` as well.

Reply:
Yes, that can be in different PR as these two are unavailable for `Generate Tests(Beta)`.
@ivikash Correct me on this!